docker/Dockerfile

```diff
@@ -62,8 +62,11 @@ RUN mkdir /var/farmOS \
   && rm -r /opt/drupal && cp -rp /var/farmOS /opt/drupal
 
 # Set the entrypoint.
-COPY docker-entrypoint.sh /usr/local/bin/
-RUN chmod u+x /usr/local/bin/docker-entrypoint.sh
+COPY --chown=www-data docker-entrypoint.sh /usr/local/bin/
+
+# Run as wwww-data user
```

Comment:
Nit: change from `wwww-data` to `www-data`

Reply:
Also please add a period at the end of the comment. There is one other comment that needs a period too apart from this one. Can you fix both please and make sure there aren't others I missed?